lib/matplotlib/backend_bases.py

```diff
@@ -1407,7 +1407,7 @@ def _update_enter_leave(self):
                 try:
                     if last.inaxes is not None:
                         last.canvas.callbacks.process('axes_leave_event', last)
-                except:
+                except Exception:
```

Comment:
Are we sure about this one given the comment below it?  Raising exceptions in GUI event loops can go very wrong very quickly (aka Qt exits the process!).

Reply:
That try-except block is only there to catch exceptions from the callbacks. `KeyboardInterrupt` and `SystemExit` can happen anywhere and Qt hat to cope with this somehow. I don't see why we should catch these events if they happen by chance while the canvas callbacks are processed.